style/styles.css

```diff
@@ -0,0 +1,1101 @@
+  padding-left: 80px;
+}
+
+.c1 {
+  background: #555;
+}
+
+.c2 {
+  background: #757575;
+}
+
+.c3 {
+  background: #858585;
+}
+
+.c4 {
+  background: #959595;
+}
+
+.linkTexts h1 {
+  margin-bottom: 20px;
+  font-size: 34px;
+}
+
+.featuredBookText h1 {
+  font-size: 34px;
+  margin-bottom: 40px;
+}
+
+.latest > div > h1 {
+  font-size: 40px;
+  width: 70%;
+}
+
+.profileCon {
+  display: flex;
+  flex-direction: column;
+  justify-content: end;
+}
+
+.profileImg {
+  width: 75px;
+  border: 5px solid white;
+  border-radius: 11px;
+  vertical-align: middle;
+  box-shadow: 0 20px 40px 10px rgb(54, 52, 52, 0.8);
+  transform: rotate(-15deg);
+  transform-origin: left bottom;
+}
+
+.profileImg:hover {
+  transform: rotate(0deg);
+  box-shadow: 0 20px 500px 10px rgb(54, 52, 52, 0);
+}
+
+.profileName {
+  font-size: 1.4em;
+  text-align: center;
+}
+
+.profileImageDiv {
+  margin-top: 20px;
+}
+
+.imgContainer {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+}
+
+.imgContainer div {
+  padding: 10px;
+  text-align: center;
+}
+
+.spanContainer {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+}
+
+.spanContainer span {
+  margin-right: 40px;
+}
+
+.fa-long-arrow-right {
+  margin-top: 70px;
+  margin-left: 0;
+}
+
+.inline {
+  display: inline-block;
+  font-size: 12px;
+}
+
+/* Articles Section */
+.wholeArticlesSection {
+  background: #f6f3f2;
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+}
+
+.articlesContainer {
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(405px, 1fr));
+  width: 1850px;
+  padding-top: 50px;
+  font-family: "Balsamiq Sans", cursive;
+}
+
+.articlesDes {
+  transition: all 0.2s ease-in-out;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  padding: 30px 30px 0 30px;
+}
+
+.articlesGra {
+  width: 80%;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  padding: 70px 70px 0 70px;
+  height: 790px;
+}
+
+.gramarly {
+  width: 326px;
+  height: 300px;
+  background: hsl(166, 65%, 82%);
+  border-radius: 11px;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+  color: grey;
+}
+
+.gramrBlock {
+  display: block;
+  text-align: center;
+}
+
+.gramarly p {
+  display: block;
+  text-align: center;
+  width: 300px;
+  margin-left: 40px;
+  font-size: 19px;
+}
+
+.gramarButton {
+  background: rgb(44, 216, 150);
+  width: 150px;
+  height: 40px;
+  margin-top: 30px;
+  border: none;
+  color: white;
+}
+
+.gramarlyPtop {
+  display: block;
+  text-align: center;
+  font-size: 13px;
+}
+
+.gramIcon {
+  width: 300px;
+}
+
+.articlesDes h2 {
+  font-size: 30px;
+}
+
+.articlesDes p {
+  font-size: 25px;
+  color: grey;
+}
+
+.articlesDes a {
+  color: rgb(71, 68, 68);
+  margin-left: 5px;
+}
+
+.redArrow {
+  text-align: center;
+  width: 150px;
+  color: rgb(158, 19, 19);
+  padding-bottom: 30px;
+  padding-top: -20px;
```

Comment:
No `padding` property accepts negative values. This is invalid.

Reply:
Changes implemented.